docs/rapids-udfs.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: page
+title: RAPIDS-Accelerated User-Defined Functions
+nav_order: 8
+---
+# RAPIDS-Accelerated User-Defined Functions
+
+This document describes how UDFs can provide a RAPIDS-accelerated
+implementation alongside the CPU implementation, enabling the
+RAPIDS Accelerator to perform the user-defined operation on the GPU.
+
```

Comment:
Again do we want to mention the experimental UDF compiler and point users at that here?